docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -169,6 +169,36 @@ the file /etc/sawtooth/validator.toml.
 
       $ sudo systemctl restart sawtooth-validator
 
+Sawtooth Rest API Configuration
+===============================
+
+Sawtooth Rest API metrics are reported by the the sawtooth-rest-api process.
```

Comment:
Put ``sawtooth-rest-api`` in double-back-quotes.

Style guide tip: Use double-back-quotes for commands, files, processes services, files and paths, config variables, protobufs, and anything else "literal" that would occur in a file or be typed on a command line.